spec/features/course/user_profile_spec.rb

```diff
@@ -5,7 +5,8 @@
   let(:instance) { Instance.default }
 
   def exp_text(course_user)
-    I18n.t('course.users.show.experience_points_earned_html',
+    I18n.t('course.users.show.level_and_exp',
+           level: course_user.level_number,
            points: course_user.experience_points)
```

Comment:
Hmm should this be `exp` instead of `points`?

Reply:
Ah thanks and updated!